.rhcicd/clowdapp-engine.yaml

```diff
@@ -327,7 +329,7 @@ parameters:
   value: "false"
 - name: NOTIFICATIONS_EMAILS_ONLY_MODE_ENABLED
```

Comment:
`clowdapp-backend.yaml` also contains that flag. Both files should have the same flag value.

Reply:
Right, I forgot this :)